pkg/util/paramtable/quota_param.go

```diff
@@ -257,6 +258,25 @@ seconds, (0 ~ 65536)`,
 	}
 	p.MaxFlushRate.Init(base.mgr)
 
+	p.MaxFlushRatePerCollection = ParamItem{
+		Key:          "quotaAndLimits.flushRate.collection.max",
+		Version:      "2.3.4",
+		DefaultValue: "0.1",
+		Formatter: func(v string) string {
+			if !p.FlushLimitEnabled.GetAsBool() {
+				return max
+			}
+			// [0 ~ Inf)
+			if getAsInt(v) < 0 {
+				return max
+			}
+			return v
+		},
+		Doc:    "qps, every 10s, rate for flush at collection level",
```

Comment:
The description here can be revised. If the rate is altered, the number 10 will become ambiguous in this context.